data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json

```diff
@@ -73,5 +73,180 @@
     "symbol": "O",
     "color": "white_magenta",
     "use_action": { "type": "learn_spell", "spells": [ "self_healing" ] }
+  },
+  {
+    "id": "blindfold_artifact",
+    "type": "ARMOR",
+    "copy-from": "blindfold",
+    "looks_like": "blindfold",
+    "name": { "str": "red devil blindfold" },
+    "description": "A red fabric blindfold with a cartoonish devil over the eyes covering tied over the eyes to block sight.  You feel some anger when you put it on.  Did that devil just wink at you?",
+    "use_action": { "type": "learn_spell", "spells": [ "spell_rage" ] }
+  },
+  {
+    "id": "artifact_heart",
+    "type": "GENERIC",
+    "category": "artifacts",
+    "name": { "str": "mother of pearl heart" },
+    "description": "A human heart grown in mother of pearl.  It beats loudly when you raise it to your ear.",
+    "weight": "310 g",
+    "volume": "345 ml",
+    "longest_side": "15 cm",
+    "price": 1000,
+    "price_postapoc": 10,
+    "material": [ "forged_dreamstuff" ],
+    "symbol": "o",
+    "color": "red",
+    "use_action": { "type": "learn_spell", "spells": [ "spell_stamina_wonder" ] }
+  },
+  {
+    "id": "artifact_eye_right",
+    "type": "GENERIC",
+    "category": "artifacts",
+    "name": { "str": "right eye" },
+    "description": "Tiny human eye.  You can clearly see the green iris and red capillaries.  You feel it can see you in the most darkest places, which is so weird since it's only an eye.",
```

Comment:
```suggestion
    "description": "Tiny human eye.  You can clearly see the green iris and red capillaries.  You feel it can see you in the darkest places, which is so weird since it's only an eye.",
```